pkg/gateway/operations/getobject.go

```diff
@@ -117,7 +117,7 @@ func (controller *GetObject) Handle(w http.ResponseWriter, req *http.Request, o
 	}
 
 	if redirect {
-		preSignedURL, _, err := o.BlockStore.GetPreSignedURL(ctx, objectPointer, block.PreSignModeRead)
+		preSignedURL, _, err := o.BlockStore.GetPreSignedURL(ctx, objectPointer, block.PreSignModeRead, entry.Path)
```

Comment:
why is this needed, what happened before?

Reply:
you are asking why do we need to provide presigned with content disposition for the gateway? it is not required - I'll remove the file path.